code/renderers/vue3/src/render.ts

```diff
@@ -16,18 +17,17 @@ export const render: ArgsStoryFn<VueRenderer> = (props, context) => {
   return () => h(Component, props, getSlots(props, context));
 };
 
-// set of setup functions that will be called when story is created
-const setupFunctions = new Set<(app: App, storyContext?: StoryContext<VueRenderer>) => void>();
-/** add a setup function to set that will be call when story is created a d
- *
- * @param fn
- */
-export const setup = (fn: (app: App, storyContext?: StoryContext<VueRenderer>) => void) => {
-  setupFunctions.add(fn);
+export const setup = (fn: (app: App, storyContext?: StoryContext<VueRenderer>) => unknown) => {
+  globalThis.PLUGINS_SETUP_FUNCTIONS ??= new Set();
+  globalThis.PLUGINS_SETUP_FUNCTIONS.add(fn);
```

Comment:
@ndelangen I thought you mentioned at some point that we prefer `@storybook/global` over `globalThis` but I don't know the reasons anymore.

Reply:
as i know globalThis was introduced to give access to global scope regardless the system Node/Browser,  it is consistency and standard  rather than using window / global

Reply:
and that  what @storybook/global  provides the global scope so if we do 
`declare var STORYBOOK_ENV:'type'` in `type.d.ts` file
will be safely available on globalThis  as global module